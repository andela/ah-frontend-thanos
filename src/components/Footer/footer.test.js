import React from 'react';
import { shallow, mount } from 'enzyme';
import { Footer, CreateImage } from './index';

describe('Footer tests', () => {
  const footer = shallow(<Footer />);
  const createImageFooter = shallow(<CreateImage />);
  const wrapper = mount(<Footer />);
  test('testing footer', () => {
    expect(footer).toMatchSnapshot();
    expect(createImageFooter).toMatchSnapshot();
    expect(wrapper.find('.footer').hasClass('row'));
  });
});
