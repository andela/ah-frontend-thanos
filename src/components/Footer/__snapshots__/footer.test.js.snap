// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer tests testing create image 1`] = `
<div
  className="image-card"
  key="[object Object]"
>
  <img
    alt="article image1"
    src={Object {}}
  />
  <div
    className="carousel-caption d-none d-md-block"
  />
</div>
`;

exports[`Footer tests testing footer 1`] = `
<div
  className="footer"
>
  <div
    className="carousel"
  >
    <div
      className="carousel slide"
      data-ride="carousel"
      id="carouselExampleIndicators"
    >
      <ol
        className="carousel-indicators"
      >
        <li
          className="active"
          data-slide-to="0"
          data-target="#carouselExampleIndicators"
        />
        <li
          data-slide-to="1"
          data-target="#carouselExampleIndicators"
        />
        <li
          data-slide-to="2"
          data-target="#carouselExampleIndicators"
        />
      </ol>
      <div
        className="carousel-inner"
      >
        <div
          className="carousel-item active"
        >
          <div
            className="row"
          >
            <div
              className="image-card"
              key="https://picsum.photos/200"
            >
              <img
                alt="article image1"
                src="https://picsum.photos/200"
              />
              <div
                className="carousel-caption d-none d-md-block"
              />
            </div>
            <div
              className="image-card"
              key="https://picsum.photos/201"
            >
              <img
                alt="article image1"
                src="https://picsum.photos/201"
              />
              <div
                className="carousel-caption d-none d-md-block"
              />
            </div>
            <div
              className="image-card"
              key="https://picsum.photos/202"
            >
              <img
                alt="article image1"
                src="https://picsum.photos/202"
              />
              <div
                className="carousel-caption d-none d-md-block"
              />
            </div>
            <div
              className="image-card"
              key="https://picsum.photos/203"
            >
              <img
                alt="article image1"
                src="https://picsum.photos/203"
              />
              <div
                className="carousel-caption d-none d-md-block"
              />
            </div>
            <div
              className="image-card"
              key="https://picsum.photos/204"
            >
              <img
                alt="article image1"
                src="https://picsum.photos/204"
              />
              <div
                className="carousel-caption d-none d-md-block"
              />
            </div>
            <div
              className="image-card"
              key="https://picsum.photos/205"
            >
              <img
                alt="article image1"
                src="https://picsum.photos/205"
              />
              <div
                className="carousel-caption d-none d-md-block"
              />
            </div>
            <div
              className="image-card"
              key="https://picsum.photos/206"
            >
              <img
                alt="article image1"
                src="https://picsum.photos/206"
              />
              <div
                className="carousel-caption d-none d-md-block"
              />
            </div>
            <div
              className="image-card"
              key="https://picsum.photos/207"
            >
              <img
                alt="article image1"
                src="https://picsum.photos/207"
              />
              <div
                className="carousel-caption d-none d-md-block"
              />
            </div>
          </div>
        </div>
      </div>
      <a
        className="carousel-control-prev"
        data-slide="prev"
        href="#carouselExampleIndicators"
        role="button"
      >
        <span
          aria-hidden="true"
          className="carousel-control-prev-icon"
        />
        <span
          className="sr-only"
        >
          Previous
        </span>
      </a>
      <a
        className="carousel-control-next"
        data-slide="next"
        href="#carouselExampleIndicators"
        role="button"
      >
        <span
          aria-hidden="true"
          className="carousel-control-next-icon"
        />
        <span
          className="sr-only"
        >
          Next
        </span>
      </a>
    </div>
  </div>
  <div
    className="footer-bottom"
  >
    <div
      className="copyright"
    >
      <span>
        Â© 2018 Authors Haven
      </span>
    </div>
    <div
      className="go-to-top"
    >
      <span>
        Top
      </span>
    </div>
    <div
      className="nav-bottom"
    >
      <ul>
        <li>
          <a
            href="/"
          >
            HOME
          </a>
        </li>
        <li>
          <a
            href="/article"
          >
            ARTICLE
          </a>
        </li>
        <li>
          <a
            href="/contact"
          >
            CONTACT
          </a>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
